modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -69,8 +69,12 @@ ifndef::openshift-dedicated,openshift-rosa[]
 * Does not support offline snapshots or resize. Volume must be attached to a running pod.
 --
 endif::openshift-dedicated,openshift-rosa[]
-
+ifndef::openshift-rosa[]
 [IMPORTANT]
 ====
 If your CSI driver is not listed in the preceding table, you must follow the installation instructions provided by your CSI storage vendor to use their supported CSI features.
 ====
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+In addition to the drivers listed in the preceding table, ROSA functions with CSI drivers from third-party storage vendors such as AWS FSX or Pure Storage Portworx. Red Hat does not oversee third-party provisioners or the connected CSI drivers and the vendors fully control source code, deployment, operation, and Kubernetes compatibility. These volume provisioners are considered customer-managed and the respective vendors are responsible for providing support. See the link:https://docs.openshift.com/rosa/rosa_architecture/rosa_policy_service_definition/rosa-policy-responsibility-matrix.html#rosa-policy-responsibilities_rosa-policy-responsibility-matrix[Shared responsibilities for Red{nbsp}Hat OpenShift Service on AWS] matrix for more information. 
```

Comment:
```suggestion
In addition to the drivers listed in the preceding table, ROSA functions with CSI drivers from third-party storage vendors such as AWS FSX or Pure Storage Portworx. Red Hat does not oversee third-party provisioners or the connected CSI drivers and the vendors fully control source code, deployment, operation, and Kubernetes compatibility. These volume provisioners are considered customer-managed and the respective vendors are responsible for providing support. See the link:https://docs.openshift.com/rosa/rosa_architecture/rosa_policy_service_definition/rosa-policy-responsibility-matrix.html#rosa-policy-responsibilities_rosa-policy-responsibility-matrix[Shared responsibilities for {product-title}] matrix for more information. 
```

Reply:
Thanks @jeana-redhat! Updated! this one is good to merge whenever you get the chance